src/components/PageSectionNumbers.tsx

```diff
@@ -0,0 +1,186 @@
+import SubsectionTextHeader from './SubsectionTextHeader'
+import NumbersInfo from './NumbersInfo'
+import Columns from './Columns/Columns'
+import Column from './Columns/Column'
+
+const baseUrl = 'https://mainnet-backend.alephium.org'
+const ONE_DAY = 1000 * 60 * 60 * 24
+
+interface Props {
+  content: {
+    title: string
+    subtitle: string
+  }
+}
+
+interface Stat<T> {
+  value: T
+  isLoading: boolean
+}
+
+const statScalarDefault = { value: 0, isLoading: true }
+
+type StatScalar = Stat<number>
+type StatScalarKeys = 'hashrate' | 'circulatingSupply' | 'totalTransactions'
+
+type StatsScalarData = { [key in StatScalarKeys]: StatScalar }
+
+const PageSectionNumbers = ({ content: { title, subtitle } }: Props) => {
+  const [explorerClient, setExplorerClient] = useState<ExplorerClient>(undefined)
+  const [statsScalarData, setStatsScalarData] = useState<StatsScalarData>({
+    hashrate: statScalarDefault,
+    circulatingSupply: statScalarDefault,
+    totalTransactions: statScalarDefault
+  })
+
+  const updateStatsScalar = useCallback(
+    (key: StatScalarKeys, value: StatScalar['value']) => {
+      setStatsScalarData((prevState) => ({ ...prevState, [key]: { value, isLoading: false } }))
+    },
+    [setStatsScalarData]
+  )
+
+  useEffect(() => {
+    setExplorerClient(new ExplorerClient({ baseUrl }))
+  }, [])
+
+  useEffect(() => {
+    if (!explorerClient) return
+
+    const fetchHashrateData = async () => {
+      const now = new Date().getTime()
+      const yesterday = now - ONE_DAY
+
+      const { data } = await explorerClient.charts.getChartsHashrates({
+        fromTs: yesterday,
+        toTs: now,
+        'interval-type': 'hourly'
+      })
+
+      if (data && data.length > 0) updateStatsScalar('hashrate', Number(data[0].value))
+    }
+
+    const fetchAndUpdateStatsScalar = async (
+      key: StatScalarKeys,
+      fetchCall: () => Promise<HttpResponse<string | number>>
+    ) => {
+      await fetchCall()
+        .then((res) => res.text())
+        .then((text) => updateStatsScalar(key, parseInt(text)))
+    }
+
+    fetchHashrateData()
+    fetchAndUpdateStatsScalar('circulatingSupply', explorerClient.infos.getInfosSupplyCirculatingAlph)
+    fetchAndUpdateStatsScalar('totalTransactions', explorerClient.infos.getInfosTotalTransactions)
+  }, [explorerClient, updateStatsScalar])
+
+  const { hashrate, circulatingSupply, totalTransactions } = statsScalarData
+  const [hashrateInteger, hashrateDecimal, hashrateSuffix] = formatNumberForDisplay(hashrate.value, 'hash')
+  const supply = formatNumberForDisplay(circulatingSupply.value)
+
+  const columns = [
+    {
+      value: 16,
+      isLoading: false,
+      description: 'shards running'
+    },
+    {
+      value: `${addApostrophes(hashrateInteger)}${hashrateDecimal ?? ''}`,
+      isLoading: false,
+      description: `${hashrateSuffix}H/s`
+    },
+    {
+      value: supply,
+      isLoading: false,
+      description: 'alph circulating'
```

Comment:
Looks weird with lower case:

<img width="1218" alt="image" src="https://user-images.githubusercontent.com/1579899/172136972-275572fb-5712-40a6-b63a-813edd2ebae5.png">

```suggestion
      description: 'ALPH circulating'
```